app/models/cloud_subnet.rb

```diff
@@ -55,6 +55,22 @@ def self.class_by_ems(ext_management_system)
     ext_management_system && ext_management_system.class::CloudSubnet
   end
 
+  def create_cloud_subnet_queue(userid, options = {})
```

Comment:
This actually needs to live in the `NetworkManager` https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/network_manager.rb